includes/admin.form.inc

```diff
@@ -14,5 +14,27 @@ function islandora_marcxml_admin_settings(array $form, array &$form_state) {
     '#description' => t("This provides an extra option to be available under the 'edit' link present on objects that have MODS datastreams."),
     '#default_value' => variable_get('islandora_marcxml_allow_editing_of_existing_mods', FALSE),
   );
+  $form['islandora_marcxml_select_marc_to_mods_xsl'] = array(
+    '#type' => 'select',
+    '#title' => t('MARC to MODS XSL file'),
+    '#default_value' => 'MARC21slim2MODS3-6.xsl',
+    '#options' => array(
```

Comment:
So the "old" included files are no longer choices? Typically I'd think they'd be the default choice.

Reply:
The old files are still there, but the newest edition of the same schema is the new default.

Reply:
Also, the spirit of this ticket is to update the transforms to the most current ones, which I believe should be the default behaviour. I added the option to select the outdated transforms as sort of a bonus feature because some sites don't like to change things -- I believe the majority would want the latest version by default, rather than having to change configuration to choose it.

Reply:
Note also that in past updates to Official LoC XSLs, we have just straight up updated them -- rather than providing the choice to keep using the old ones.

Everything has been addressed as far as I see. Are you able to approve?